src/shared/install.c

```diff
@@ -56,6 +56,9 @@
 
 #define UNIT_FILE_FOLLOW_SYMLINK_MAX 64
 
+#define PRESET_ENABLE  1
+#define PRESET_DISABLE 2
+
```

Comment:
https://github.com/systemd/systemd/pull/3143#discussion_r61515248

I'm partial to an enum as well.